### DISCUSSION TITLE
add patterns to the command palette

### DISCUSSION THREAD

**Discussion by chrisgrieser** [jglev/obsidian-apply-patterns-plugin#14] [Ideas]
Patterns could be added directly to the command palette. This way, you can assign a hotkey to it and apply patterns with only one click :)

The Quickadd Plugin has such a feature where you can add individual quickadds to the command palette.

**Comment by jglev** [comment]
Thank you for writing!

I agree that it would be useful to be able to run commands from the Command Palette and hotkeys. I drafted new functionality for this just [today](https://github.com/publicus/obsidian-apply-patterns-plugin/discussions/5#discussioncomment-1159933); it's now [in line](https://github.com/publicus/obsidian-apply-patterns-plugin/pull/11) for inclusion in the next version of the plugin. 

Again, thank you!

**Reply by chrisgrieser** [reply]
wow, many thanks 🤗
